src/einsteinpy/tests/test_metric/test_schwarzschild.py

```diff
@@ -59,12 +59,17 @@ def test_calculate_trajectory(
     )
     _c, _scr = constant.c.value, schwarzschild_radius(M).value
     ans = ans[1]
-    testarray = (
-        (1 - (_scr / ans[:, 1])) * np.square(ans[:, 4])
-        - (np.square(ans[:, 5])) / ((1 - (_scr / ans[:, 1])) * (_c ** 2))
-        - np.square(ans[:, 1] / _c)
-        * (np.square(ans[:, 6]) + np.square(np.sin(ans[:, 2])) * np.square(ans[:, 7]))
-    )
+    testarray = list()
+    for ansi in ans:
+        g = schwarzschild_utils.metric(_c, ansi[1], ansi[2], _scr)
+        tmp = (
```

Comment:
Instead of naming it tmp, can we name it more intuitively?

Reply:
@shreyasbapat Could you have a look at the code and suggest something.

Reply:
same to be applied to `test_kerr.py`

Reply:
maybe remove `tmp` altogether and instead do `testarray.append( whole value )`. Same for `test_kerr.py`. Remember about black and isort

Reply:
> 
> 
> maybe remove `tmp` altogether and instead do `testarray.append( whole value )`. Same for `test_kerr.py`. Remember about black and isort

should i do this ?

Reply:
Yes! Go ahead!

Reply:
It's still better than naming variables "tmp" :)